packages/react-utilities/src/hooks/useIsomorphicLayoutEffect.ts

```diff
@@ -1,5 +1,5 @@
 import * as React from 'react';
-import { isSSR } from '../utils/index';
+import { isSSR } from '../ssr/index';
```

Comment:
what do you think about using `useIsSSR` instead that also reads from context in addition to the `document` and `window` checks ?

Reply:
I was thinking about, but then we will still need a function for cases like this:

```tsx
export const useIsomorphicLayoutEffect: typeof React.useEffect = isSSR() ? React.useEffect : React.useLayoutEffect;
```

Reply:
what if `isSSR` stays internal to utilities ? I can't imagine components really needing the pure function over context

Reply:
I am thinking about pure `canUseDOM()` that can be used in utils and `useIsSSR()` for components? WDYT?

Reply:
Nice, that would remove the double SSR with `useIsSSR` I approve 👍👍

Reply:
Done in bf6fb05.